### PR TITLE
kNN bench, configs and runner update

### PR DESCRIPTION
**Pull request by Alexsandruss**
- Separate training and prediction in kNN benchmarks, add unsupervised support
- Add configs for sklearn, cuML and XGBoost
- Control daal4py sklearn patching by FORCE_DAAL4PY_SKLEARN env variable
- Support of multiple configs in runner

### CODE REVIEW

File: cuml/bench.py [hunk 1]
```diff
@@ -8,6 +8,18 @@
 import sklearn
 import timeit
 import json
+import os
+import sys
+
+
+if 'FORCE_DAAL4PY_SKLEARN' in os.environ.keys():
```
Comment (PetrovKP) [inline cuml/bench.py:15]: Add it to Readme

Comment (PetrovKP) [inline cuml/bench.py:15]: Similar to sklearn patch
```suggestion
if os.environ.get('FORCE_DAAL4PY_SKLEARN', False) in [True, '1', 'y', 'yes', 'Y', 'YES', 'Yes', 'true', 'True', 'TRUE']:
```

File: cuml/bench.py [hunk 2]
```diff
@@ -8,6 +8,18 @@
 import sklearn
```
Comment (PetrovKP) [inline cuml/bench.py:8]: do we really need to copy this file to cuml/sklearn/xgboost - given that it is the same everywhere?

File: configs/cuml_config.json [hunk 3]
```diff
@@ -0,0 +1,486 @@
+{
```
Comment (PetrovKP) [inline configs/cuml_config.json:1]: I would put it in folders:
- sklearn
- xgboost
- cuml
...

File: cuml/bench.py (hunk 2 above)
Comment (Alexsandruss) [reply cuml/bench.py:8]: Not in scope of this PR

File: configs/cuml_config.json (hunk 3 above)
Comment (Alexsandruss) [reply configs/cuml_config.json:1]: Not in scope of this PR

File: cuml/bench.py (hunk 1 above)
Comment (Alexsandruss) [reply cuml/bench.py]: Added

Comment (Alexsandruss) [reply cuml/bench.py]: Added

**Pull request merged**